packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3436,55 +3553,60 @@ class SemanticsConfiguration {
     return true;
   }
 
-  /// Absorb the semantic information from `other` into this configuration.
+  /// Absorb the semantic information from `child` into this configuration.
   ///
   /// This adds the semantic information of both configurations and saves the
   /// result in this configuration.
   ///
+  /// The [RenderObject] owning the `child` configuration must be a descendant
+  /// of the [RenderObject] that owns this configuration.
+  ///
   /// Only configurations that have [explicitChildNodes] set to false can
   /// absorb other configurations and it is recommended to only absorb compatible
   /// configurations as determined by [isCompatibleWith].
-  void absorb(SemanticsConfiguration other) {
+  void absorb(SemanticsConfiguration child) {
```

Comment:
this might deserve a breaking change announcement

Reply:
Done: https://groups.google.com/forum/#!topic/flutter-announce/Y1TG0CNgsM8